lib/src/resizable_controller.dart

```diff
@@ -142,7 +142,7 @@ class ResizableController with ChangeNotifier {
       throw ArgumentError('Size cannot exceed total available space.');
     }
 
-    if (resizableSizes.totalRatio > 1.0) {
+    if (resizableSizes.totalRatio > 1.01) {
```

Comment:
Is this absolutely necessary? 

Are there any side-effects of upping this constraint? What if the total ratio is 1.009? 

I haven't tested, but this seems like it could cause an overflow.

Reply:
It is necessary to have some sort of small epsilon value, as the ratios can often sum to a number slightly bigger than 1, thanks to floating point math. This is true even when using the numbers provided by `.ratios` directly. 

The epsilon could be made smaller, but I did not run into issues despite hitting this code path often

Reply:
Maybe I can look into some kind of "clamping" logic if there are any issues in the future.